distributed/utils.py

```diff
@@ -1211,11 +1211,28 @@ def reset_logger_locks():
                 is_kernel_and_no_running_loop = True
 
         if not is_kernel_and_no_running_loop:
-            import tornado.platform.asyncio
 
-            asyncio.set_event_loop_policy(
-                tornado.platform.asyncio.AnyThreadEventLoopPolicy()
-            )
+            BaseEventLoopPolicy = asyncio.DefaultEventLoopPolicy
+            if WINDOWS and sys.version_info >= (3, 8):
+                # WindowsProactorEventLoopPolicy is not compatible with tornado 6
+                # fallback to the pre-3.8 default of Selector
+                # https://github.com/tornadoweb/tornado/issues/2608
```

Comment:
From what I understand, tornado decided not to do this for users by default. Did Jupyter decided whether or not to do it?

Is appropriate for Dask to me making this decision?

Reply:
Tornado decided not to set the `asyncio.WindowsSelectorEventLoopPolicy()` by default and instead recommends projects do that themselves. Jupyter and several other projects have done this (e.g. https://github.com/jupyter/notebook/pull/5047).

Our situation is a little different because we want to set the event loop policy to tornado's `AnyThreadEventLoopPolicy` to let us create loops inside of any thread. Tornado has decided to set the base class for `AnyThreadEventLoopPolicy` to be `asyncio.WindowsSelectorEventLoopPolicy` on Windows (https://github.com/tornadoweb/tornado/commit/d2af6a6fbe57a4d3bc06fbde6f4c1f4e8ec06dd3), but there hasn't been a release with the those changes yet so we're doing that ourselves here. I'll add a note that we can remove this once the next tornado release is out.